bluepysnap/frame_report.py

```diff
@@ -240,7 +240,7 @@ def data_units(self):
     @property
     def node_set(self):
         """Returns the node set for the report."""
-        return self.simulation.node_sets[self.to_libsonata.cells]
+        return self.simulation.node_sets.content[self.to_libsonata.cells]
```

Comment:
Still open to discussion if this should just return the node set name.

Usage would then be:
```python
sim.circuit.get(report.node_set, node_sets=sim.node_sets)
```

Reply:
From what I see, `cells` is optional at https://sonata-extension.readthedocs.io/en/latest/sonata_simulation.html#reports

> Specify which node_set to report, default is the simulation “node_set”.

I suppose that returning the name should be enough, but it's better to wait for other opinions.